src/test/java/com/google/api/codegen/testdata/discoveries/go/go_compute.v1.json.baseline

```diff
@@ -4090,13 +4090,13 @@ func main() {
     // TODO: Handle error.
   }
 
-  // Project ID for this request.
+  // * Project ID for this request.
```

Comment:
Yes, this inconsistency is on my fix list, but not in scope for this PR, which introduces minimal consistent changes to fix compilecheck.